interpreter/istio/src/main/scala/io/buoyant/interpreter/k8s/istio/IstioInterpreterInitializer.scala

```diff
@@ -32,6 +33,9 @@ case class IstioInterpreterConfig(
   @JsonIgnore
   override val experimentalRequired = true
 
+  @JsonIgnore
+  val _ = Logger.get(this.getClass.getName).warning("Istio K8S Interpreter has been deprecated since version 1.4.7")
```

Comment:
do you need to assign this to a dummy val?  Can you just do `Logger.get(...).warning(...)`?

Reply:
Ahh, it looks like we don't. I thought that since this was a function of type `Unit` we would need to capture the expression in a dummy variable to avoid the `discarded Unit` error. TIL.